packages/neos-ui-editors/src/Reference/index.js

```diff
@@ -1,65 +1,63 @@
-import React, {PureComponent, PropTypes} from 'react';
+import React, {Component, PropTypes} from 'react';
 import {connect} from 'react-redux';
-import {$transform, $get} from 'plow-js';
+import {$transform} from 'plow-js';
 import SelectBox from '@neos-project/react-ui-components/lib/SelectBox/';
-import {neos} from '@neos-project/neos-ui-decorators';
-
 import backend from '@neos-project/neos-ui-backend-connector';
+import {selectors} from '@neos-project/neos-ui-redux-store';
+import {neos} from '@neos-project/neos-ui-decorators';
 
 @connect($transform({
-    activeWorkspace: $get('cr.workspaces.personalWorkspace.name'),
-    siteNodePath: $get('cr.nodes.siteNode')
+    contextForNodeLinking: selectors.UI.NodeLinking.contextForNodeLinking
 }))
 @neos(globalRegistry => ({
     i18nRegistry: globalRegistry.get('@neos-project/neos-ui-i18n')
 }))
-export default class ReferenceEditor extends PureComponent {
+export default class ReferenceEditor extends Component {
```

Comment:
Is it necessary to let the Editor inherit from `Component`? We should definitely stick to `PureComponent` otherwise